src/ray/raylet/worker_pool.cc

```diff
@@ -44,20 +46,21 @@ WorkerPool::~WorkerPool() {
   pool_.clear();
   actor_pool_.clear();
   registered_workers_.clear();
-  started_worker_pids_.clear();
+  starting_worker_processes_.clear();
 }
 
 uint32_t WorkerPool::Size() const {
   return static_cast<uint32_t>(actor_pool_.size() + pool_.size());
 }
 
-void WorkerPool::StartWorker(bool force_start) {
+void WorkerPool::StartWorkerProcess(bool force_start) {
   RAY_CHECK(!worker_command_.empty()) << "No worker command provided";
   // The first condition makes sure that we are always starting up to
   // num_cpus_ number of processes in parallel.
-  if (NumWorkersStarting() > num_cpus_ && !force_start) {
+  if (static_cast<int>(starting_worker_processes_.size()) > num_cpus_ && !force_start) {
```

Comment:
should "static_cast<int>(starting_worker_processes_.size()) > num_cpus_" be >= here? When the ==, after the processing, the process count will be greater than num_cpus_.

Reply:
good catch